.github/workflows/release.yaml

```
@@ -0,0 +1,27 @@
+name: goreleaser
+
+on:
+  pull_request:
+  push:
```

Nit, to be consistent with other files here:
```suggestion
on: [pull_request, push]
```